sdk/test-utils/recorder/package.json

```diff
@@ -99,6 +98,46 @@
     "rimraf": "^3.0.0",
```

Comment:
Let's upgrade to latest `rimraf` which then you need to put in `rimraf --glob` in the usage.